demo/src/screens/componentScreens/SliderScreen.tsx

```diff
@@ -56,170 +65,229 @@ export default class SliderScreen extends Component<SliderScreenProps, SliderScr
-
-          <View row spread centerV marginB-10>
-            <Text $textDefault text70BO>
-              Default slider
-            </Text>
-            <Button link label="Reset" onPress={this.resetSlider}/>
-          </View>
-          
-          <View row centerV style={this.getReverseStyle()}>
-            <Icon assetName={'search'} style={styles.image}/>
-            <Slider
-              ref={this.slider}
-              onValueChange={this.onSliderValueChange}
-              value={INITIAL_VALUE}
-              minimumValue={0}
-              maximumValue={100}
-              step={1}
-              containerStyle={styles.sliderContainer}
-              disableRTL={forceLTR}
-            />
-            <Text bodySmall $textNeutral style={styles.text} numberOfLines={1}>
-              ${sliderValue}
-            </Text>
-          </View>
-
-          <Text $textDefault text70BO marginT-30>
-            Negatives
-          </Text>
-          <Slider
-            minimumValue={-100}
-            maximumValue={100}
-            value={-30}
-            minimumTrackTintColor={Colors.red30}
-            thumbTintColor={Colors.red50}
-            containerStyle={styles.slider}
-          />
-          <Slider
-            minimumValue={-300}
-            maximumValue={-100}
-            value={-130}
-            minimumTrackTintColor={Colors.red30}
-            thumbTintColor={Colors.red50}
-            containerStyle={styles.slider}
-          />
-
-          <Text $textDefault text70BO marginT-20>
-            Disabled
-          </Text>
-          <Slider minimumValue={100} maximumValue={200} value={120} containerStyle={styles.slider} disabled/>
 
-          <Text $textDefault text70BO marginT-15>
-            Custom with Steps
-          </Text>
-          <Slider
-            value={50}
-            minimumValue={0}
-            maximumValue={100}
-            step={25}
-            containerStyle={styles.slider}
-            trackStyle={styles.track}
-            thumbStyle={styles.thumb}
-            activeThumbStyle={styles.activeThumb}
-            thumbTintColor={Colors.violet70}
-            minimumTrackTintColor={Colors.violet40}
-            maximumTrackTintColor={Colors.violet70}
-          />
+  renderDefaultSliderExample() {
+    const {sliderValue, forceLTR} = this.state;
 
-          <Text $textDefault text70BO marginV-15>
-            Range Slider
+    return (
+      <Fragment>
+        <View row spread centerV marginB-10>
+          <Text $textDefault text70BO>
+            Default slider
           </Text>
-          <View row spread style={this.getReverseStyle()}>
-            <Text bodySmall $textNeutral>
-              min. {sliderMinValue}%
-            </Text>
-            <Text bodySmall $textNeutral>
-              max. {sliderMaxValue}%
-            </Text>
-          </View>
+          <Button link label="Reset" onPress={this.resetSlider}/>
```

Comment:
The `Reset` button resets the presented value but not the slider's value

Reply:
Fixed